src/style.css

```diff
@@ -0,0 +1,156 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  color: #2a2a2a;
+  font-family: 'Lato', sans-serif;
+}
+
+.to-do-container {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  width: 1200px;
+  max-width: 40%;
+  margin: 200px auto;
+  list-style: none;
+  justify-items: center;
+  border-radius: 5px;
+  box-shadow: 1px 3px 20px -8px #000;
+}
+
+.to-do-container > * {
+  border-bottom: 1px solid rgb(185, 185, 185);
+  padding: 0.7rem;
```

Comment:
Please stick to value maybe it's rem or px, but you using both in padding properties for best practices.

Reply:
am happy you enjoy my code
Thanks for the feedback 😄 @